region/crateria/central/Parlor and Alcatraz.json

```
@@ -1122,24 +1146,98 @@
     },
     {
       "link": [5, 8],
-      "name": "Ceiling Clip Alcatraz Escape",
+      "name": "Grapple X-Ray Ceiling Clip Alcatraz Escape",
```

```suggestion
      "name": "Alcatraz Escape with Grapple X-Ray Ceiling Clip",
```
The name of the room should go first. This is one option, but it could be named however you'd like